doc/source/changes/version_0_28.rst.inc

```diff
@@ -1,7 +1,63 @@
-﻿New features
+﻿Backward incompatible changes
+-----------------------------
+
+* changed behavior of operators `session1 == session2` and `session1 != session2`: returns a boolean
+  (closes:issue:`516`):
+
+    >>> s1 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> s2 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> # equivalent to all(s1.equals(s2))
+    >>> s1 == s2
+    True
+    >>> s2.arr1['a1'] = 0
+    >>> s1 == s2
+    False
+    >>> # equivalent to any(~s1.equals(s2))
+    >>> s1 != s2
+    True
+    >>> if s1 == s2:
+    ...     print("session s1 is equal to session s2")
+    ... else:
+    ...     print("session s1 is not equal to session s2:\\n{}".format(s1.equals(s2)))
+    session s1 is not equal to session s2:
+    name   arr1   arr2   arr3
+          False  False  False
+
+  Use `Session.equals` to get the previous behavior.
+
+
+New features
 ------------
 
-* added a feature (see the :ref:`miscellaneous section <misc>` for details).
+* added method `equals` to `Session` object to compare arrays from two sessions (closes:issue:`517`):
+
+    >>> s1 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> s2 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> s1.equals(s2)
```

Comment:
I **_wonder_**, if we should not rather return a single value (True) like Axis.equals. I know this is not what I wrote in the issue (my bad!), but the general idea in making those changes is to make a Session more like an LArray with one special "array" axis, and given #518, it would make more sense to return a single value. Now to get the old s1 == s2 behaviour, one would need to write: `(s1 == s2).all_by('array')`, but this is not implemented yet and possibly a bit cumbersome to type, so we might want to make Session.equals(other) return a bool and create another method (e.g. Session.array_equals or something like that) which would have the old == behavior. Thoughts?